src/Crypto/WebAuthn/Model.hs

```diff
@@ -927,7 +939,13 @@ data CollectedClientData (t :: WebauthnKind) raw = CollectedClientData
     ccdCrossOrigin :: Bool,
     -- | Raw data of the client data, for verification purposes
     ccdRawData :: RawField raw
-    -- TODO: Implement this
+    -- TODO: This library does not implement token binding, this is in
+    -- anticipation of version 3 of the webauthn spec that likely removes this
+    -- field in its entirety. Discussion can be found in
+    -- [the relevant PR](https://github.com/w3c/webauthn/pull/1630).
+    -- Chromium and Firefox both already no longer propagate this field.
```

Comment:
"no longer" implies that support for this existed at some point, which I don't think is the case